src/libs/ValidationUtils.js

```diff
@@ -206,6 +206,30 @@ function meetsAgeRequirements(date) {
     return testDate.isValid() && testDate.isBetween(oneHundredFiftyYearsAgo, eighteenYearsAgo);
 }
 
+/**
+ * Validate that given date is in a specified range of years before now.
+ *
+ * @param {String} date
+ * @param {Number} minimumAge
+ * @param {Number} maximumAge
+ * @returns {String}
+ */
+function getAgeRequirementError(date, minimumAge, maximumAge) {
+    const recentDate = moment().subtract(minimumAge, 'years');
+    const longAgoDate = moment().subtract(maximumAge, 'years');
+    const testDate = moment(date);
+    if (!testDate.isValid()) {
+        return Localize.translateLocal('privatePersonalDetails.error.dateInvalid');
+    }
+    if (testDate.isBetween(longAgoDate, recentDate)) {
+        return '';
+    }
```

Comment:
This is the only "valid" date case - I wonder if it would be clearer if this was at the bottom of this function?